engine/utils/Macros.h

```diff
@@ -24,4 +24,7 @@
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
 
+// Abort command
+#define EXIT_APP abort();
```

Comment:
Leave a note to turn this into a function somewhere at some point since being a macro doesn't really provide any additional value from being a macro.

Reply:
Removed